doc/api/net.md

```diff
@@ -501,7 +507,30 @@ added: v0.5.3
 
 The amount of bytes sent.
 
-### socket.connect(options[, connectListener])
+### socket.connect()
+
+Initiate the connection on a given socket.
+
+Normally this method is not needed, the socket should be created and
+opened with [`net.createConnection()`][]. Use this only if you are
+implementing a custom Socket.
+
+Possible signatures:
+
+* [socket.connect(options[, connectListener])][`socket.connect(options)`]
+* [socket.connect(path[, connectListener])][`socket.connect(path)`]
+  for connections to UNIX domain sockets or Windows named pipes.
```

Comment:
"for IPC connections" (since they can be used for RPC as well)? Also good idea on defining what IPC mechanism Node.js supports in one place and then linking to it.